### ISSUE TITLE
The coeff example in energy model

### ISSUE THREAD

**Issue by Rutsow** [harvard-edge/airlearning-ue4#5]
Hi,
I have succeed in running it. The distanceTraveled and flighttime works well in th UE4 windows but the energy not. I find that the coeffs of the energy model are 0. Would you mind show an example of the coeffs?
Thanks
![image](https://user-images.githubusercontent.com/49801486/88890406-2bbbf900-d274-11ea-9a0e-bc55a984a687.png)

**Comment by aqibsaeed** [comment]
Hi @Rutsow 

Could you share the steps you took to install AirSim, I am unable to generate AirSim plugin.

**Comment by pdx97** [comment]
> Hi @Rutsow
> 
> Could you share the steps you took to install AirSim, I am unable to generate AirSim plugin.

Yeah i am also facing the same issue while generating Airsim Plugin

**Comment by qinglong0276** [comment]
> 嗨，我已经成功运行了它。距离旅行和飞行时间在UE4窗口中运行良好，但能量却不行。我发现能量模型的系数为 0。你介意举一个系数的例子吗？ 谢谢 ![image](https://user-images.githubusercontent.com/49801486/88890406-2bbbf900-d274-11ea-9a0e-bc55a984a687.png)

我运行的过程出现**编译错误的问题,**有三个蓝图加载不进去,并且我是全部按照步骤安装,可是我的**无法生成 AirSim 插件**,我应该怎么办
